mlflow/tracking/fluent.py

```diff
@@ -596,7 +597,7 @@ def get_parent_run(run_id: str) -> Optional[Run]:
     return MlflowClient().get_parent_run(run_id)
 
 
-def log_param(key: str, value: Any) -> Any:
+def log_param(key: str, value: Any, synchronous: bool = True) -> Union[RunOperations, ParamValue]:
```

Comment:
Same here, is `ParamValue` required?

Reply:
Talked on the slack, making this wrapper inline in each file.